### PR TITLE
Complete README

### PR DESCRIPTION
**Pull request by cprovencher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,108 @@
...
 
-(work in progress)
+## How can I see a high-level view of what we support?
+1. ```git clone https://github.com/dcos/dcos-images.git ~/dcos-images```
+2. ```brew install tree``` or ```apt install tree```
+3. ```cd ~/dcos-images```
+4. ```tree```
+
+There's also a [DC/OS Platform Version Compatibility Matrix](https://docs.mesosphere.com/version-policy/#dcos-platform-version-compatibility-matrix)
+
```
Comment (orsenthil) [inline README.md]: Please add a very high level description of this project.

It should answer - How to can I add a new version of docker, a new pre-requisite and quality it for DC/OS.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,108 @@
...
+When images are built, jenkins will commit them back to the pull request as a dcos_images.yaml file. Jenkins will commit
+those images right after they are built, after the cluster successfully launches, after the tests run successfully or
+never, depending on your publish_and_test_config.yaml. That commit will trigger a new jenkins build because of the
+webhook, but the Jenkinsfile will detect it and exit immediately with success. We don't want to lose the link to the real
+job that is still running the tests or creating a cluster, so jenkins will also comment on the pull request the link to
+that job. Depending on the results of that job, it will be up to you to judge if the results are good enough to merge
+the pull request and new DC/OS images in the repo.
+
+To modify the following chart, go to [draw.io](https://www.draw.io/) and import draw-io-diagram.xml (located at the root level of this repo)
+![flow-chart](flow-diagram.png)
```
Comment (orsenthil) [inline README.md]: Please move this diagram before the introduction.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,108 @@
...
+- **desired_cluster_profile.tfvars**  
+Configuration for the terraform cluster that will be created using new DC/OS images. This cluster will be used to install
+and test DC/OS with the new images.
+
+- **publish_and_test_config.yaml**  
+Determines what is required for newly-built images to be commited back to a pull request by jenkins. Also determines
+what tests are ran for on a pull request. [See this example](https://github.com/dcos/dcos-images/blob/master/oracle-linux/7.4/aws/DCOS-1.11.3/docker-1.13.1/publish_and_test_config.yaml#L1).
+You can specify the field 'publish_dcos_images_after' with possible values: 'packer_build', 'dcos_installation',
+'integration_tests' and 'never'. For example, if I want jenkins to push the newly-built DC/OS AMIs to my pull request as
+soon as they are built by packer, I would specify 'packer_build'. If I want more confidence and want to make sure
```
Comment (orsenthil) [inline README.md]: Do not use `I want` , `I` etc in the README.  Write the entire document in the 3rd person ( https://en.wikiversity.org/wiki/Talk:Technical_writing_style)

File: README.md [hunk 4]
```diff
@@ -1,8 +1,108 @@
...
+Packer configuration to build DC/OS images. Base images and install_dcos_prerequisites.sh are some of its inputs.
+
+- **desired_cluster_profile.tfvars**  
+Configuration for the terraform cluster that will be created using new DC/OS images. This cluster will be used to install
+and test DC/OS with the new images.
+
+- **publish_and_test_config.yaml**  
+Determines what is required for newly-built images to be commited back to a pull request by jenkins. Also determines
+what tests are ran for on a pull request. [See this example](https://github.com/dcos/dcos-images/blob/master/oracle-linux/7.4/aws/DCOS-1.11.3/docker-1.13.1/publish_and_test_config.yaml#L1).
+You can specify the field 'publish_dcos_images_after' with possible values: 'packer_build', 'dcos_installation',
```
[the remark this answers is not in the document]
Comment (margaret) [reply README.md]: https://www.tablesgenerator.com/markdown_tables :parrot:

File: README.md (hunk 3 above)
Comment (cprovencher) [reply README.md]: @orsenthil What you linked specifically mentions not to write in the 3rd person. It says to use 2nd person.

File: README.md [hunk 5]
```diff
@@ -1,8 +1,108 @@
...
+never, depending on your publish_and_test_config.yaml. That commit will trigger a new jenkins build because of the
+webhook, but the Jenkinsfile will detect it and exit immediately with success. We don't want to lose the link to the real
+job that is still running the tests or creating a cluster, so jenkins will also comment on the pull request the link to
+that job. Depending on the results of that job, it will be up to you to judge if the results are good enough to merge
+the pull request and new DC/OS images in the repo.
+
+To modify the following chart, go to [draw.io](https://www.draw.io/) and import draw-io-diagram.xml (located at the root level of this repo)
+![flow-chart](flow-diagram.png)
+
+__*1__: install_dcos_prerequisites.sh is directly referenced inside of packer.json
```
[the remark this answers is not in the document]
Comment (cprovencher) [reply README.md]: @orsenthil This is not an enumeration, but rather footnotes that provide help for the diagram

**Pull request merged**